content/en/docs/Writing policies/match-exclude.md

```diff
@@ -121,6 +121,28 @@ spec:
 Although the above snippet is useful for showing the types of matching that you can use, most policies either use one or just a couple different elements within their `match` statements.
 {{% /alert %}}
 
+### Match a Deployment deployed in a Namespace having a specific label
+
+This is an example that selects a Deployment deployed in a namespce. Where the namespace is with a label `app=connector` or `app=compute`.
```

Comment:
first misspelling in "Namespace" (Namespace is initially capitalized)

Reply:
"This example selects all new Deployments destined for a Namespace with labels `app=connector` or `app=compute`.